python/mp/rosary.py

```diff
@@ -4,13 +4,14 @@
 import threading
 import time
 import math
+import inspect
 
-from mp import color
-from .effects import *
 from pythonosc import udp_client
 from pythonosc import osc_bundle_builder
 from pythonosc import osc_message_builder
 
+from mp import color, effects
```

Comment:
And further down, I need to pass mp.effects somehow to my `register_defined_effects` method, instead of just having effects.* imported.